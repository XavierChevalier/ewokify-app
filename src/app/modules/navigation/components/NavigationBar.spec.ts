import { generateStorybookSnapshotTests } from '@/tests/tools/StorybookSnapshot'
import { Default } from '@/app/modules/navigation/components/NavigationBar.stories'
import { ComponentTestsGenerator } from '@/tests/tests-generators/ComponentTestsGenerator'
import NavigationBar from './NavigationBar.vue'

describe('NavigationBar', () => {
  generateStorybookSnapshotTests({ Default })

  const componentTestsGenerator = new ComponentTestsGenerator(NavigationBar)
  componentTestsGenerator.itShouldBeDefined()
})
