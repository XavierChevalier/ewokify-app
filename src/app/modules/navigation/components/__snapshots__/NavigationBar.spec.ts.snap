// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`NavigationBar [snapshot] should render the "Default" component correctly 1`] = `<div />`;
